src/main.rs

```diff
@@ -118,10 +118,17 @@ fn run_remove_sequence(args: Args) -> Result<(), Error> {
 	proton_cli::remove_sequence(&admin_key_path, &name)
 }
 
-#[allow(unused_variables)]
 fn run_list_permissions(args: Args) -> Result<(), Error> {
-	let permissions = proton_cli::get_permissions();
-	Ok(println!("{}", permissions.join("\n")))
+	let private_key = args.arg_private_key;
+	if private_key.is_some() {
+		let _ = proton_cli::get_permissions(&private_key.unwrap())
+			.map(|p| println!("{:?}", p));
```

Comment:
What does this debug print actually look like? Can you show an example output, exactly as it would be printed?